portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Raul's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="topbar">
+      <ul>
+          <li class="topbartext">Raul Cruz</li>
+          <li class="logo"><a href="https://github.com/raulcruise" target="_blank"><img src="/images/github.png" alt="GitHub logo" style="right:25px;"></a></li>
```

Comment:
Let's use newlines for every block.

ex:
<li ...>
  <a ...>
    <img ...>

Reference:
https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

Reply:
Done.